draft-ietf-core-conditional-attributes.md

```diff
@@ -201,7 +201,9 @@ bool notifiable( Resource * r ) {
 #define LT_EX ( r->v < r->lt ^ r->last_rep_v < r->lt )
 #define GT_EX ( r->v > r->gt ^ r->last_rep_v > r->gt )
 #define ST_EX ( abs( r->v - r->last_rep_v ) >= r->st )
-#define IN_BAND ( ( r->gt <= r->v && r->v <= r->lt ) || ( r->lt <= r->gt && r->gt <= r->v ) || ( r->v <= r->lt && r->lt <= r->gt ) )
+#define IN_BAND ( ( r->gt <= r->v && r->v <= r->lt ) ||
+                  ( r->lt <= r->gt && r->gt <= r->v ) ||
```

Comment:
C requires backslash-escaping continuation lines of #defines...

Reply:
Very true, that was my mistake!